httpie.yaml

```diff
@@ -15,7 +15,10 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - py3-multidict
       - py3-pip
+      - py3-pygments
```

Comment:
You might need to add these as runtime dependencies

Reply:
I can do that after the test complete.  Thanks for pointing out where to add them. 🙏🏻